include/cantera/kinetics/Falloff.h

```diff
@@ -22,24 +22,49 @@ class AnyMap;
 
 /**
  * Base class for falloff function calculators. Each instance of a subclass of
- * Falloff computes one falloff function. This base class implements the
- * trivial falloff function F = 1.0.
- *
- * @ingroup falloffGroup
+ * Falloff computes one falloff function.
  */
 class Falloff
 {
 public:
-    Falloff() {}
+    Falloff()
+        : allow_negative_pre_exponential_factor(false)
+        , third_body_concentration(NAN)
+        , rate_index(npos)
+        , m_chemicallyActivated(false)
+        , m_rc_low(NAN)
+        , m_rc_high(NAN)
+    {
+        m_work.resize(workSize());
+    }
+
     virtual ~Falloff() {}
 
     /**
      * Initialize. Must be called before any other method is invoked.
      *
      * @param c Vector of coefficients of the parameterization. The number and
      *     meaning of these coefficients is subclass-dependent.
+     *
+     * @todo  deprecate; superseded by setData
+     */
+    void init(const vector_fp& c);
+
+    /**
+     * Set coefficients of the falloff parametetrization.
```

Comment:
```suggestion
     * Set coefficients of the falloff parameterization.
```
Likewise below a few lines.